src/client.ts

```diff
@@ -604,6 +604,25 @@ export class UXDClient {
     });
   }
 
+  public createSetMangoDepositoryQuoteMintAndRedeemSoftCapInstruction(
+    softCap: number,
+    controller: Controller,
+    depository: MangoDepository,
+    authority: PublicKey,
+    options: ConfirmOptions
+  ): TransactionInstruction {
+    const softCapNativeBN = new BN(
+      new BN(softCap).mul(new BN(10).pow(new BN(depository.quoteMintDecimals)))
+    );
+    return this.instruction.setMangoDepositoryQuoteMintAndRedeemSoftCap(softCapNativeBN, {
+      accounts: {
```

Comment:
When the key have the same name as the variable, you can do:

```
     accounts: {
        authority,
        controller: controller.pda,
      },
      options,
```

this is called `shorthand`

Reply:
Fixed! (also have this in Rust no excuse hehe)